homeassistant/components/landisgyr_heat_meter/sensor.py

```diff
@@ -0,0 +1,108 @@
+"""Platform for sensor integration."""
+from __future__ import annotations
+
+from dataclasses import asdict
+import logging
+
+from homeassistant.components.sensor import (
+    ATTR_STATE_CLASS,
+    RestoreSensor,
+    SensorDeviceClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_DEVICE_CLASS, ATTR_UNIT_OF_MEASUREMENT
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+from homeassistant.util import dt as dt_util
+
+from . import DOMAIN
+from .const import GJ_TO_MWH, HEAT_METER_SENSOR_TYPES
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the sensor platform."""
+    _LOGGER.info("The Landis+Gyr Heat Meter sensor platform is being set up!")
```

Comment:
Please don't log at info level. Setting up a platform is normally not something we need to log.